src/main/java/ui/MenuControl.java

```diff
@@ -336,9 +334,11 @@ private MenuItem createRefreshMenuItem() {
     private void updateRepoRemoveList(Menu remove) {
         remove.getItems().clear();
 
-        Set<String> currentlyUsedRepos = ui.getCurrentlyUsedRepos();
-        Set<String> removableRepos = ui.logic.getStoredRepos();
-        removableRepos.removeAll(currentlyUsedRepos);
+        Set<String> currentlyUsedRepos = ui.getCurrentlyUsedRepos()
+                .stream().map(String::toLowerCase).collect(Collectors.toSet());
+        Set<String> removableRepos = ui.logic.getStoredRepos()
+                .stream().filter(repoId -> !currentlyUsedRepos.contains(repoId.toLowerCase()))
+                .collect(Collectors.toSet());
```

Comment:
So `getCurrentlyUsedRepos` does not normalize the names to lower cases as we would like to show it as what the user has input. Just a few lines below this we will call non-normalize `getCurrentlyUsedRepos`. `toLowerCase` for `repoId` is called (instead of mapped) because we would like to use the original `repoId` casing instead of the normalized one as per the reason mentioned.